ebos/eclcpgridvanguard.hh

```diff
@@ -280,6 +283,27 @@ protected:
             equilGrid_.reset(new Dune::CpGrid(*grid_));
             equilCartesianIndexMapper_.reset(new CartesianIndexMapper(*equilGrid_));
         }
+
+
+#ifdef ENABLE_3DPROPS_TESTING
+        std::vector<int> actnum;
+        unsigned long actnum_size;
+        if (mpiRank == 0) {
+            actnum = Opm::UgGridHelpers::createACTNUM(*grid_);
+            actnum_size = actnum.size();
+        }
+
+#ifdef HAVE_MPI
+        MPI_Bcast(&actnum_size, 1, MPI_UNSIGNED_LONG, 0, MPI_COMM_WORLD);
```

Comment:
please use `grid().comm().bcast(&actnum_size, 1, 0)``then you can skip the ifdef